### PR TITLE
Updated the README, security policy, issue templates, and other details

### PR DESCRIPTION
**Pull request by DougTidwell**
As discussed with the team earlier, I made the following changes: 

* Updated the README file
* Updated the security policy (SECURITY.md)
* Updated the python script that generates  SECURITY.md. Not sure why we need to generate that file, but the original repo did that. Data format of the input to the supported versions table changed also. 
* Cleaned up and streamlined the issue templates. 
* Deleted every instance of the ClickHouse logo.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/15_bug-report.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+**Installation issue** - something doesn't install they way it should
+
+**Usability issue** - something works, but it could be a lot easier
+
+**Documentation issue** - something in the docs is wrong, incomplete, or confusing
+
+## Describe the situation
+A clear, concise description of what's happening. 
+
```
Comment (hodgesrm) [inline .github/ISSUE_TEMPLATE/15_bug-report.md:33]: Something to add when discussion reproduction:  Does it reproduce in a ClickHouse Official build of the same version?

**Review by hodgesrm — APPROVED** [review]
Approved with addition of change to issue template (does it reproduce in ClickHouse Official build?)

File: .github/ISSUE_TEMPLATE/15_bug-report.md (hunk 1 above)
Comment (Enmk) [reply .github/ISSUE_TEMPLATE/15_bug-report.md:33]: Can we also request a minimum viable scenario/reproduction script?

**Review by Enmk — APPROVED** [review]
LGTM

**Pull request merged**